index.html

```diff
@@ -6,121 +6,135 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.css">
-    <title>JQuery Target Practice</title>
-
+    <title>JavaScript Target Practice</title>
+    <style>
+        #assignments > * {
+            margin-top: 3rem;
+            padding-top: 1rem;
+            border-top: 1px solid lightgrey;
+        }
+
+        #assignments > *:first-child {
+            border-top: none;
+        }
+    </style>
 </head>
 
 <body>
     <div class="container">
-        <div class="jumbotron">
-            <h1>JQuery Target Practice!</h1>
-        </div>
-
-        <div class="row well">
-            <h3>Instructions</h3>
-            <p>This page contains a bunch of html elements for you to play with.</p>
-            <p>Each
-                <code> section </code> has instructions for you to follow.</p>
-            <p>In order to complete all the exercises, you will need to refer to
-                <a href="https://api.jquery.com/">JQuery Documentation</a> and inspect the page</p>
+        <div class="jumbotron text-center">
+            <h1>JavaScript Target Practice!</h1>
         </div>
 
-        <div id="set-col-3" class="row well">
-            <div>
-                <h4>Use JQuery to select this div and add col-lg-3.</h4>
+        <div class="card">
+            <h2 class="card-header">Instructions</h2>
+            <div class="card-body">
+                <p>This page contains a bunch of html elements for you to play with.</p>
+                <p>Each
+                    <code> section </code> has instructions for you to follow.</p>
+                <p>In order to complete all the exercises, you will need to refer to
+                    <a href="https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector">
+                        documentation for selecting DOM Elements</a>.</p>
+                <p>Edit <code>targetPractice.js</code> to solve the problems.</p>
             </div>
         </div>
 
-        <div id="background-swap" class="row well">
-            <div class="col-lg-6" style="background: blue; color: white">
-                <h4>Change this background to be red</h4>
+        <div id="assignments">
+            <div id="set-col-3" class="mt-4">
+                <h3>Use JavaScript to select this div and add the class <code>col-3.</code></h3>
             </div>
-            <div class="col-lg-6" style="background: red; color: white">
-                <h4>Change this background to be blue</h4>
+
+            <div id="background-swap" class="row">
```

Comment:
For these were you thinking about using [setAttribute](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute)?

Reply:
Oh good idea. Maybe specify using a class. But I do think that a direct inline style manipulation with JS could be a good exercise too.